core-primitives/attestation-handler/src/collateral.rs

```diff
@@ -0,0 +1,154 @@
+/*
+	Copyright 2022 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+use sgx_types::sgx_ql_qve_collateral_t;
+use std::{io::Write, string::String, vec::Vec};
+
```

Comment:
```suggestion
use crate::sgx_reexport_prelude::serde_json;
use sgx_types::sgx_ql_qve_collateral_t;
use std::{io::Write, string::String, vec::Vec};
```

Reply:
I think it is needed, or at least for me it was, because it would not compile.

Reply:
Indeed. I also fixed it.